test/testsupport/testdata/TdAll.scala

```diff
@@ -268,66 +312,76 @@ object TdAll {
+          )
         ),
-        Instalment(
-          instalmentNumber          = InstalmentNumber(1),
-          dueDate                   = DueDate(LocalDate.parse("2022-08-28")),
-          instalmentInterestAccrued = InterestAccrued(AmountInPence(3)),
-          instalmentBalance         = InstalmentBalance(AmountInPence(111141)),
-          debtItemChargeId          = ChargeReference("A00000000001"),
-          amountDue                 = AmountDue(AmountInPence(55570)),
-          debtItemOriginalDueDate   = DebtItemOriginalDueDate(LocalDate.parse("2021-07-28"))
+        instalments          = List(
+          Instalment(
+            instalmentNumber          = InstalmentNumber(2),
+            dueDate                   = DueDate(LocalDate.parse("2022-09-28")),
+            instalmentInterestAccrued = InterestAccrued(AmountInPence(3)),
+            instalmentBalance         = InstalmentBalance(AmountInPence(55571)),
+            debtItemChargeId          = ChargeReference("A00000000001"),
+            amountDue                 = AmountDue(AmountInPence(55570)),
+            debtItemOriginalDueDate   = DebtItemOriginalDueDate(LocalDate.parse("2021-07-28"))
+          ),
+          Instalment(
+            instalmentNumber          = InstalmentNumber(1),
+            dueDate                   = DueDate(LocalDate.parse("2022-08-28")),
+            instalmentInterestAccrued = InterestAccrued(AmountInPence(3)),
+            instalmentBalance         = InstalmentBalance(AmountInPence(111141)),
+            debtItemChargeId          = ChargeReference("A00000000001"),
+            amountDue                 = AmountDue(AmountInPence(55570)),
+            debtItemOriginalDueDate   = DebtItemOriginalDueDate(LocalDate.parse("2021-07-28"))
+          )
         )
-      )
-    ),
-    customerDetails             = customerDetails,
-    regimeDigitalCorrespondence = regimeDigitalCorrespondence
-  )
+      ),
+      customerDetails             = customerDetails,
+      regimeDigitalCorrespondence = regimeDigitalCorrespondence
+    )
+  }
 
   def customerDetail(email: String = "bobross@joyofpainting.com", source: EmailSource = EmailSource.ETMP): Option[List[CustomerDetail]] =
     Some(List(CustomerDetail(Some(Email(SensitiveString(email))), Some(source))))
 
   val someRegimeDigitalCorrespondenceFalse: Option[RegimeDigitalCorrespondence] = Some(RegimeDigitalCorrespondence(false))
   val someRegimeDigitalCorrespondenceTrue: Option[RegimeDigitalCorrespondence] = Some(RegimeDigitalCorrespondence(true))
 
-  val arrangementResponse: ArrangementResponse = ArrangementResponse(ProcessingDateTime("2022-03-23T13:49:51.141Z"), CustomerReference("123PA44545546"))
+  def arrangementResponse(taxRegime: TaxRegime): ArrangementResponse = {
+    val customerReference = taxRegime match {
+      case TaxRegime.Epaye => CustomerReference("123PA44545546")
+      case TaxRegime.Vat   => CustomerReference("123PA44545546")
```

Comment:
Should the VAT customer ref be `101747001` just for consistency (seen that one used below)? I don't mind though

Reply:
fixed [here](https://github.com/hmrc/essttp-frontend/pull/264/commits/c7d07634174ecb39cffd22bfc025cd0fbab42c7e)